orchestration/IntegrationService/Implementations/AzureResourceManagerDeploymentService.ps1

```diff
@@ -53,17 +53,32 @@ Class AzureResourceManagerDeploymentService: IDeploymentService {
                             [string] $deploymentTemplate, `
                             [string] $deploymentParameters, `
                             [string] $location) {
-
-        # call arm validation
-        $validation = `
-            $this.InvokeARMOperation(
-                $tenantId,
-                $subscriptionId,
-                $resourceGroupName,
-                $deploymentTemplate,
-                $deploymentParameters,
-                $location,
-                "validate");
+        
+        # Try to fetch the validation resource group
+        $validationResourceGroup = `
+            Get-AzResourceGroup `
+                -Name $resourceGroupName `
+                -ErrorAction SilentlyContinue;
+
+        # Does the validation resource group exists?
+        if($null -ne $validationResourceGroup) {
+            # call arm validation
+            $validation = `
+                $this.InvokeARMOperation(
+                    $tenantId,
+                    $subscriptionId,
+                    $resourceGroupName,
+                    $deploymentTemplate,
+                    $deploymentParameters,
+                    $location,
+                    "validate");
+        }
+        else {
+            # Fail early if the validation resource group does not
+            # exists
+            Throw "Validation resource group not setup. Create the validation resource `
```

Comment:
Add the name of the validation resource group as part of the error message.

Reply:
Updated.